dll/win32/shell32/CCopyToMenu.cpp

```diff
@@ -45,7 +45,9 @@ HRESULT _GetCidlFromDataObject(IDataObject *pDataObject, CIDA** ppcida)
 CCopyToMenu::CCopyToMenu() :
     m_idCmdFirst(0),
     m_idCmdLast(0),
-    m_idCmdCopyTo(-1)
+    m_idCmdCopyTo(-1),
+    m_fnOldWndProc(NULL),
+    m_bInSettingText(FALSE)
```

Comment:
What is this flag for?

Reply:
For checking/locking text update status. See `WindowProc` and `BrowseCallbackProc`.

Reply:
If `m_bInSettingText` is TRUE, then it is in processing of `EN_CHANGE` or `BFFM_SELCHANGED`.

Reply:
Renamed the variable as `m_bIgnoreTextBoxChange` and simplified the code.